src/pypi_changes/_print/__init__.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Iterable
+
+from .._cli import Options
+from .._pkg import Package
+
+
+class Reversor:
```

Comment:
This probably could/should be private so named _Reversor? or nested within the method below?

Reply:
This was already in the original code.

Reply:
Done